Source/Core/Core/IOS/ES/ES.h

```diff
@@ -81,6 +81,12 @@ class ES final : public Device
     s32 ipc_fd = -1;
   };
 
+  enum class CheckContentHashes : bool
```

Comment:
Silly question - what's the point of this deriving from bool?

Reply:
It's not strictly necessary, but I think this makes it very obvious that this enum class is just a glorified boolean.